.travis.yml

```diff
@@ -60,7 +58,13 @@ jobs:
           env: DMD=2.070.2.s* F=production
         - <<: *test-matrix
           env: DMD=2.070.2.s* F=devel
-
+        - stage: Generate deployment image
+          env: DMD=2.070.2.s* F=production
+          script:
+              - BEAVER_DOCKER_IMG=builder BEAVER_DOCKER_CONTEXT=docker/builder beaver dlang install
+              - BEAVER_DOCKER_IMG=builder beaver dlang make
+              - BEAVER_DOCKER_IMG=builder beaver dlang make pkg
+              - BEAVER_DOCKER_IMG=sociomantictsunami/dhtnode BEAVER_DOCKER_CONTEXT=docker/runner beaver install
```

Comment:
Same here. You can then only override the special cases.

Reply:
@leandro-lucarella-sociomantic btw how expensive would it be to use artifact cache to pass deb packages here on each build (and not just tags)?

Reply:
If it's needed, it should be fine. Definitely better than rebuilding! :)